launchers/VirSat Core Build Development (Deploy Unsecured).launch

```diff
@@ -14,6 +14,9 @@
     <stringAttribute key="M2_USER_SETTINGS" value=""/>
     <booleanAttribute key="M2_WORKSPACE_RESOLUTION" value="false"/>
     <booleanAttribute key="org.eclipse.debug.core.ATTR_FORCE_SYSTEM_CONSOLE_ENCODING" value="false"/>
+    <mapAttribute key="org.eclipse.debug.core.environmentVariables">
+        <mapEntry key="GITHUB_SHA" value="12345ABCDEF"/>
```

Comment:
This looks like a placeholder value?

Reply:
Yes, the build scripts (in maven pom) are using it for release builds. If oyu want to test run a release build locally, it is better to have it set.